## osu! fix: reverts Proton commit breaking alt-tab on certain window managers
diff --git a/dlls/winex11.drv/window.c b/dlls/winex11.drv/window.c
index db51152e1dc..872b1c8c7a7 100644
--- a/dlls/winex11.drv/window.c
+++ b/dlls/winex11.drv/window.c
@@ -1339,15 +1339,7 @@ void update_net_wm_states( struct x11drv_win_data *data )
         new_state |= (1 << NET_WM_STATE_MAXIMIZED);
 
     ex_style = NtUserGetWindowLongW( data->hwnd, GWL_EXSTYLE );
-    if ((ex_style & WS_EX_TOPMOST) &&
-        /* This workaround was initially targetting some mutter and KDE issues, but
-         * removing it causes failure to focus out from exclusive fullscreen windows.
-         *
-         * Many games do not have any specific logic to get out of exclusive fullscreen
-         * mode, and we have currently no way to tell exclusive fullscreen from a window
-         * with topmost + fullscreen styles, so we cannot properly implement it either.
-         */
-        !(new_state & (1 << NET_WM_STATE_FULLSCREEN)))
+    if (ex_style & WS_EX_TOPMOST)
         new_state |= (1 << NET_WM_STATE_ABOVE);
     if (!data->add_taskbar)
     {
